examples/maven-multiproject/pom.xml

```diff
@@ -3,12 +3,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.saveourtool.diktat</groupId>
-    <artifactId>diktat-examples-maven</artifactId>
+    <artifactId>diktat-examples-maven-multiproject</artifactId>
     <packaging>pom</packaging>
-    <version>1.2.6-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
suggestion to put just 0.0-SNAPSHOT since it's just a stub for version

Reply:
Done